test/e2e/authorizer/authorizer_test.go

```diff
@@ -151,32 +105,48 @@ func TestAuthorizer(t *testing.T) {
 	require.Equal(t, len(f.Servers), 1, "incorrect number of servers")
 
 	server := f.Servers["main"]
+
 	kcpCfg, err := server.Config()
 	require.NoError(t, err)
-	clusterName, err := framework.DetectClusterName(kcpCfg, ctx, "workspaces.tenancy.kcp.dev")
-	require.NoError(t, err)
-	kubeClients, err := kubernetes.NewClusterForConfig(kcpCfg)
+	kubeClusterClient, err := kubernetes.NewClusterForConfig(kcpCfg)
 	require.NoError(t, err)
-	kcpClients, err := kcp.NewClusterForConfig(kcpCfg)
+	kcpClusterClient, err := kcp.NewClusterForConfig(kcpCfg)
 	require.NoError(t, err)
-	kcpClient := kcpClients.Cluster(clusterName)
+	dynamicClusterClient, err := dynamic.NewClusterForConfig(kcpCfg)
 	require.NoError(t, err)
-	dynamicClient, err := dynamic.NewForConfig(kcpCfg)
+
+	orgClusterName := framework.NewOrganizationFixture(t, f.Servers["main"])
+	_, org, err := helper.ParseLogicalClusterName(orgClusterName)
 	require.NoError(t, err)
 
+	kubeClient := kubeClusterClient.Cluster(orgClusterName)
+	kcpClient := kcpClusterClient.Cluster(orgClusterName)
+	dynamicClient := dynamicClusterClient.Cluster(orgClusterName)
+
 	clients := map[string]*clients{
-		"admin": {
-			KubeClient: kubeClients.Cluster(clusterName),
+		"org": {
+			KubeClient: kubeClient,
 			KcpClient:  kcpClient,
 			Dynamic:    dynamicClient,
 		},
-		"user-1": newUserClient(t, "user-1", "admin", "workspace1", kcpCfg),
-		"user-2": newUserClient(t, "user-2", "admin", "workspace1", kcpCfg),
-		"user-3": newUserClient(t, "user-3", "admin", "workspace1", kcpCfg),
+		"user-1": newUserClient(t, "user-1", helper.EncodeOrganizationAndWorkspace(org, "workspace1"), kcpCfg),
```

Comment:
@sttts is it possible to contribute these e2e test changes as a separate PR? I'd love to reuse those for me extended e2e tests.

Reply:
done